assignments/objects.js

```diff
@@ -19,26 +19,81 @@ const example = {
 
 // Write your intern objects here:
 
+let internMitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
+
+let internKennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function(){
+    return "Hello, my name is Kennan!"
+  }
+}
+
+let internKeven = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+}
+
+let internGannie = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M"
+}
+
+let internAntonietta = {
+  id: 5,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function(num1, num2){
+    return num1*num2
+  }
+}
+
+
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(internMitzi.name);
+
 // Kennan's ID
 
+console.log(internKennan.id);
+
 // Keven's email
 
+console.log(internKeven.email);
+
 // Gannie's name
 
+console.log(internGannie.name);
+
 // Antonietta's Gender
 
+console.log(internAntonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+console.log(internKennan.speak());
```

Comment:
You could also define the functions inline if you wanted:
```js
internKennan.speak = function() {
  return "Hello, my name is Kennan!"
}
```
You could also refer to kennan's name here by also using the `this` keyword. What `this` refers to inside the function is the `internKennan` object itself.
```js
internKennan.speak = function() {
  return `Hello, my name is ${this.name}`;
}
```